### PR TITLE
Rework plugin broker to send broker execution result to Che-master over JSON_RPC instead of providing a REST API

### PR DESCRIPTION
**Pull request by garagatyi**
This PR:
- Removes usage of REST API (but REST API itself is not yet removed in the PR for the time being)
- Start Che plugins resolving process immediately after the start of the plugin broker 
- Fetches Che plugins config from a file
- Connects to the Che master by JSON_RPC over websocket
- After start/successful/unsuccessful execution sends events to JSON_RPC with the status, error, result of execution
- Add fixes to the tar processing code
- Updates dependencies of the project
- Adds more logs to understand on which step failure appeared

### CODE REVIEW

File: broker/broker.go [hunk 1]
```diff
@@ -13,32 +13,129 @@
...
+	})
+}
+
+// PushStatuses sets given tunnel as consumer of broker events.
+func PushStatuses(tun *jsonrpc.Tunnel) {
+	bus.SubAny(&tunnelBroadcaster{tunnel: tun}, model.BrokerStatusEventType, model.BrokerResultEventType)
+}
+
+type tunnelBroadcaster struct {
+	tunnel          *jsonrpc.Tunnel
```
Comment (skabashnyuk) [inline broker/broker.go]: everything except tunnel is unused in this struct. Is this intentionally?

**Comment by garagatyi** [comment]
I've aligned commit messages with the PR description and pushed with force to the PR. Sorry about that.

File: broker/broker.go (hunk 1 above)
Comment (garagatyi) [reply broker/broker.go]: It was copied by mistake. Reconnection feature is not implemented in the broker. I've removed this code.

**Comment by garagatyi** [comment]
@skabashnyuk I've removed rest and changed deps.

**Comment by garagatyi** [comment]
@skabashnyuk And I've checked the broker in the walking skeleton with all those unchecked before dependencies. Everything works.

**Comment by garagatyi** [comment]
@benoitf those were debug traces, I've removed them.

**Pull request merged**